src/js/angular/core/directives/import-resource-tree/recursive.directive.js

```diff
@@ -0,0 +1,27 @@
+const modules = [];
+angular
+    .module('graphdb.framework.core.directives.recursive', modules)
+    .directive('recursive', recursive);
+
+recursive.$inject = ['$compile'];
+
+function recursive($compile) {
+    return {
+        restrict: "EACM",
+        priority: 100000,
+        compile: function (tElement, tAttr) {
+            var contents = tElement.contents().remove();
```

Comment:
I know that this code is borrowed, but can you clean it up a bit please. For example, change these `var`s to `let` or `const`, rename attributes, as we don't have the practice to use such names

Reply:
I removed the "recursive" directive. It was unused.